client/app/src/main/java/com/microsoft/research/karya/data/repo/WorkerRepository.kt

```diff
@@ -190,4 +103,8 @@ class WorkerRepository @Inject constructor(private val workerAPI: WorkerAPI, pri
         return@withContext workerDao.getById(id)
     }
 
+        suspend fun upsertWorker(worker: WorkerRecord) = withContext(Dispatchers.IO) {
+        workerDao.upsert(worker)
+    }
+
```

Comment:
This is not being used anywhere and the indentation is incorrect.

Reply:
It is being used, not in this version but my viewmodels are using it